components/textfield/index.css

```diff
@@ -131,207 +453,278 @@ governing permissions and limitations under the License.
   */
   -moz-appearance: textfield;
 
-  &::placeholder {
-    font-weight: var(--spectrum-textfield-texticon-placeholder-text-font-weight);
-    font-style: var(--spectrum-textfield-texticon-placeholder-text-font-style);
-    transition: color var(--spectrum-global-animation-duration-100) ease-in-out;
-    opacity: 1;
+  /* Remove the native clear button in IE */
```

Comment:
I cannot claim this, it came from the existing code. It only appears that I added it because of the extent of the refactoring I did in the migration. Always great to learn things from the code that others wrote!